src/routes/settings/+page.svelte

```diff
@@ -21,11 +22,12 @@
 	let ollamaModel = $settingsStore?.ollamaModel || '';
 	let ollamaTagResponse: OllamaTagResponse | null = null;
 
-	$: settingsStore.set({
+	$: settingsStore.update((settings) => ({
+		...settings,
 		ollamaServer,
 		ollamaModels: ollamaTagResponse?.models || [],
 		ollamaModel
-	});
+	}));
```

Comment:
I think this may be a left-over change, if so let's also revert it.

Reply:
You're right, but I think this change should be present in #107 because `settingsStore` will also have `userTheme`, so if you do `settingsStore.set` with only the ollama variables, the `userTheme` will get lost.

Reply:
I see, if that's the case let's fix it in #107 and remove it from this PR, that way both are easier to review.